src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1089,26 +1089,6 @@ public Long execute(Jedis connection) {
   }
 
   @Override
-  public List<String> blpop(final String arg) {
```

Comment:
This guys weren't marked as `@Deprecated` but I believe there is no problem to delete it since `JedisCommands` marked it as `@Deprecated`. :+1: